src/rapala.py

```diff
@@ -81,19 +75,24 @@ def init_driver(self) -> webdriver:
         This func initializes the webdriver and disables images
         A wait is initialized with a 5 second timeout
         """
-        options = Options()
-        options.add_argument("--headless")
-        options.add_experimental_option("prefs", self.prefs)
-        service = Service(self.chrome_path)
-        driver = webdriver.Chrome(service=service, options=options)
+        from selenium.webdriver.firefox.options import Options
+        firefox_options = Options()
+        from selenium.webdriver.chrome.options import Options
+        chrome_options = Options()
+
+        driver_dict = {"firefox_driver": webdriver.Firefox, "chrome_driver": webdriver.Chrome}
+        option_dict = {"firefox_driver": firefox_options, "chrome_driver": chrome_options}
+        service = Service(self.driver_path)
+        driver = driver_dict[self.driver_type](service=service, options=option_dict[self.driver_type])
```

Comment:
Two thing to note here

- When you import Options in 80, it replaces the options in 78 in the namespace. When there's a possibility of collision, use the pattern: "from X import Y as Z"
- It may be better to use an if/else here to decide which Options to import. Then you won't need your `options_dict` anymore. You can just initialize whichever Options was imported and pass to the webdriver.

Reply:
The if/else would only be for importing since the initialization and eventual use is same for both types

Reply:
I'm still thinking, the option_dict would still be needed, I had to create service_dict too. Pls check the updates in rapala.py to see if I should still use the if/else statement